### ISSUE TITLE
Города как объекты

### ISSUE THREAD

**Issue by findoff** [Kvisaz/nukecaravan#2]
Как насчет сделать города объектами и соотвественную проверку по ним?

Это позволит например в будущем сделать экономику города.

Например кол-во браминов, наемников и товара. И от количества ставить цены.

Если прибавить периодическое рандомное колебание, это позволит реалистично торговать без особых вычислительных затрат и усложнения кодовой базы.

**Comment by Kvisaz** [comment]
Да, это запланировано. Над механикой думаю. Это будет новый плагин, старый отключим.
